marko/ext/latex_renderer.py

```diff
@@ -0,0 +1,185 @@
+"""
+LaTeX renderer
+"""
+import logging
+from typing import List, Optional, Set, Union
+
+from marko import Renderer
+
+_logger = logging.getLogger(__name__)
+
+
+class LatexRenderer(Renderer):
+    """Render the parsed Markdown to LaTeX format."""
+
+    packages: Set[str]
+    graphic_paths: Set[str]
+
+    # Header levels that will be numbered. Default none.
+    # It could be a list of header levels (ex. [1, 2]) or header names ['part', 'section']
+    # TODO: add unit test
+    numbered_headers: List[Union[int, str]]
+
+    def __init__(self):
+        super().__init__()
+        self.packages = set()
+        self.graphic_paths = set()
+        self.numbered_headers = []
```

Comment:
Would be better to clear them inside `__enter__()` or `__exit__()`

Reply:
Done, please check.

Reply:
"graphic_paths" is removed.
I belive "numbered_headers" should treated differently, please check "test_render_headers".